digitalsynth/src/components/css/Footer.css

```diff
@@ -1,10 +1,11 @@
 .FooterNav {
-  width: 100vw;
+  width: 89vw;
   display: flex;
   flex-direction: row;
   justify-content: space-around;
   align-items: center;
   margin: 0 auto;
-  font-family: "Press Start 2P";
-  font-size: 1.3vh;
+  font-family: "Toxigenesis";
+  font-size: 1.8vh;
```

Comment:
`vh` is not a good unit for font-size
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units